python/examples/core/harness.py

```diff
@@ -38,7 +39,7 @@ class Measurements:
   config_keys = [
     "expert",
     "np_types",
-    "compile_time_problem_sizes_dict",
+    "runtime_only",
```

Comment:
I'd love to get rid of `runtime_only` and replace by `dynamic_at_compile_time` or something similarly descriptive.

Reply:
I would suggest to do that in a followup. We also wanted to name the experts. All these changes should probably be separate commits since they touch all the test_harness calls?

Reply:
SG